src/Form/Type/PixelType.php

```diff
@@ -23,6 +24,14 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                     'placeholder' => 'setono_sylius_facebook.form.pixel.pixel_id_placeholder',
                 ],
             ])
+            ->add('customAccessToken', TextareaType::class, [
```

Comment:
I think it should just be named `accessToken`

Reply:
`custom` indicates that there is some non-custom/default one exists in the universe :)

Reply:
Yeah, I still disagree, but let's keep it :)